cmd/sync.go

```diff
@@ -36,6 +37,10 @@ func init() {
 		"state", "s", []string{"kong.yaml"}, "file(s) containing Kong's configuration.\n"+
 			"This flag can be specified multiple times for multiple files.\n"+
 			"Use '-' to read from stdin.")
+	syncCmd.Flags().StringVarP(&syncWorkspace, "workspace", "w",
```

Comment:
Let's remove the `-w` shorthand flag. This flag will not be used often and `-w` will hamper readability of commands later down the road.

Reply:
Done.